<launch>
<include file="$(find yumi_description)/launch/display.launch" />
<node name="kdl_kinematics" type="kdl_kinematics" pkg="controller" />
<node name="positionControl" type="positionControl.py" pkg="controller" output="screen"/>
</launch>